src/update_categories.py

```diff
@@ -0,0 +1,66 @@
+import json
+from os import listdir, path
+from os.path import isfile, join
+import argparse
+import re
+
+"""
+Script for updating the categories, domains, and reasoning field for all the tasks.
```

Comment:
"update" is a bit open-ended. How about "simplify" or "sparsify"? 
Say a few words as to why we're doing this.